templates/navigation.html

```diff
@@ -0,0 +1,69 @@
+{% load static %}
+<link rel="stylesheet" href="{% static 'custom.css' %}">
+<nav class="navbar navbar-expand-lg bg-body-tertiary m-20">
+  <div class="container-fluid mt-5 container-margin">
+    <a class="navbar-brand" href="#">
+      <h1>RadicalS</h1>
+    </a>
+    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+      <span class="navbar-toggler-icon"></span>
+    </button>
+
+    <div class="collapse navbar-collapse w-200" id="navbarSupportedContent">
+
+      <ul class="navbar-nav mx-auto">
+        <li class="nav-item dropdown">
+          <a class="nav-link dropdown-toggle" href="#" role="button" data-bs-toggle="dropdown" aria-expanded="false">Products</a>
+          <ul class="dropdown-menu hover">
+            <li class="nav-item">
+              <a class="nav-link" href="#">Category</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#">Category1</a>
+            </li>
+          </ul>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="#">Contacts</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="#">About Us</a>
+        </li>
+      </ul>
+
+      <ul class="navbar-nav mb">
+        {% if user.is_authenticated %}
+          <li class="nav-item dropdown">
+            <a class="nav-link dropdown-toggle" href="#" role="button" data-bs-toggle="dropdown" aria-expanded="false">{{ user.first_name }}</a>
+            <ul class="dropdown-menu hover">
+              <li class="nav-item">
+                <a class="nav-link" href="#">Account</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#">Logout</a>
+              </li>
+            </ul>
+          </li>
+        {% else %}
+          <li class="nav-item">
+            <a class="nav-link" href="#">Login</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#">Register</a>
+          </li>
+        {% endif %}
+      </ul>
+
+      <ul class="navbar-nav mb">
+        <li class="nav-item">
```

Comment:
Since there is no way to tell what kind of icon it is just from looking at the code, and that issue would carry over to accessibility issues, add `aria-label` to this tag describing what is this icon.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label

Reply:
as example `<li class="nav-item" aria-lanel="Loginmenu option">`